### ISSUE TITLE
Manual

### ISSUE THREAD

**Issue by DigitalWarrior** [Brewtarget/brewtarget#797]
Version 3.0.11
OS Windows 10

When trying to open up the manual nothing happens. 
Looking at the log. 
[20:47:58.289] (9sk) WARNING : ShellExecute 'file:///C:/Program Files/Brewtarget-3.0.11/bin/../data/manual-en.pdf' failed (error 2).  [:0]

After searching C:/Program Files/Brewtarget-3.0.11/bin and every sub folder. There is no folder called data nor is there there any file called manual-en.pdf.

**Comment by matty0ung** [comment]
This is a build/packaging bug with the file not getting included.   However, I think the fix may end up being to remove the link to the manual.

There is a PDF manual in the source repository (at https://github.com/Brewtarget/brewtarget/blob/develop/doc/manual-en.pdf) but it's rather out of date and I don't know how the file was generated.  It's possible that @mikfire knows more about this.

**Comment by DigitalWarrior** [comment]
Why not update the manual? It looks like it was a word doc that was converted into a PDF. I could take a stab at updating it and submitting a pull request if you would like. No sure when I would have it done though. I was able to convert the PDF to Word
[manual-en.docx](https://github.com/user-attachments/files/15508157/manual-en.docx)

**Comment by DigitalWarrior** [comment]
I stand corrected. The manual appears to be generated from here https://github.com/Brewtarget/manual
Still willing to take a stab at updating it.

**Comment by matty0ung** [comment]
Yes, that would be brilliant.  If you can update the manual, I'll fix the build settings etc so it gets reincluded with the software. :+1:

**Comment by matty0ung** [comment]
I think used to do both.  Eg you can access the old online manual at https://brewtarget.gitbooks.io/user-manual/content/en/ (and in French at https://brewtarget.gitbooks.io/user-manual/content/fr/).  Somehow it all got generated from https://github.com/Brewtarget/manual/tree/master.

Going forwards, I'd be inclined to say we should go with something that's:
 -  online, so we don't have to do a software release to fix a documentation typo
 -  easy to edit (and easy to track changes in), so it's more likely to be kept up-to-date.

We could do something simple at https://github.com/Brewtarget/brewtarget/wiki, but I'm very open to suggestions for other options.

**Comment by DigitalWarrior** [comment]
Got gh-pages up and working. 
![image](https://github.com/Brewtarget/brewtarget/assets/802654/b69fcaaf-5f0a-4da7-a754-7ed52d8eca8d)

Just a rough layout. Have a lot to do yet.
